more_itertools/recipes.py

```diff
@@ -200,7 +204,10 @@ def roundrobin(*iterables):
     """
     # Recipe credited to George Sakkis
     pending = len(iterables)
-    nexts = cycle(iter(it).next for it in iterables)
+    if PY2:
```

Comment:
This is sort of an eyesore, but seemed better than `getattr(it, '__next__', getattr(it, 'next'))`